src/DIRAC/WorkloadManagementSystem/Service/JobManagerHandler.py

```diff
@@ -85,7 +85,7 @@ def initializeHandler(cls, serviceInfoDict):
       cls.log.warn("Cannot connect to OptimizationMind!", result['Message'])
     return S_OK()
 
-  def initialize(self):
+  def initializeRequest(self):
```

Comment:
There is still an issue that `self.diracSetup = self.serviceInfoDict['clientSetup']` doesn't exist for tornado.

Perhaps it needs to be added here like the VO is? Else I'm not sure if there was a different plan for this parameter?

https://github.com/DIRACGrid/DIRAC/blob/d8c1c45d087ac543ab99b510470c08a7bd2370ad/src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py#L474


Unless I've missed it we should also add the client version to the request as well like was recently done for DISET.